blog_project/blog/routes.py

```diff
@@ -126,4 +126,16 @@ def update(post_id):
     elif request.method == 'GET':
         form.title.data = post.title
         form.content.data = post.content
-        return render_template('update.html', form=form)
\ No newline at end of file
+        return render_template('update.html', form=form)
+
+@app.route('/post/search' , methods=['GET', 'POST'])
+def search():
+    form = SearchForm()  
+    if form.validate_on_submit():
+        posts = Post.query.filter_by(title=form.query.data).all()
```

Comment:
Thanks, but to find posts with similar titles, it would be better to be like this:
`posts = Post.query.filter(Post.title.like(f"%{form.query.data}%")).all()`

Please change it and resend.

Reply:
Sure.

Reply:
Changes applied @farhadmpr